props/saber_shtok_buttons.h

```diff
@@ -61,25 +96,34 @@ class SaberShtokButtons : public PropBase {
 
 // Turn Blade OFF	
       case EVENTID(BUTTON_POWER, EVENT_HELD_LONG, MODE_ON):
-      case EVENTID(BUTTON_AUX, EVENT_LATCH_OFF, MODE_ON):
-      case EVENTID(BUTTON_AUX2, EVENT_LATCH_OFF, MODE_ON):
-#if NUM_BUTTONS == 0
-      case EVENTID(BUTTON_NONE, EVENT_TWIST, MODE_ON):
-#endif
     if (!SaberBase::Lockup()) {
+#ifndef DISABLE_COLOR_CHANGE
+          if (SaberBase::GetColorChangeMode() != SaberBase::COLOR_CHANGE_MODE_NONE) {
+            // Just exit color change mode.
+            // Don't turn saber off.
+            ToggleColorChangeMode();
+            return true;
+          }
+#endif
       Off();
     }
         return true;
 
-// Force
+// Force and Color Change mode
       case EVENTID(BUTTON_NONE, EVENT_TWIST, MODE_ON | BUTTON_POWER):
+#ifndef DISABLE_COLOR_CHANGE
+        if (accel_.x < -0.15) {
```

Comment:
It's better to use fusor.down().x < -0.15
The down() vector handles noise and rotation better.

Reply:
I just used these lines from the original code, I don't know coding :)

Reply:
Guess I'll update it later then.